### DISCUSSION TITLE
"Export all" produces bad KML

### DISCUSSION THREAD

**Discussion by dmillman** [OpenTracksApp/OpenTracks#1398] [Q&A] [answered]
Hi everyone,

I'm trying to Export All (for archiving).

Tried:
* KMZ (with photos)
* KMZ (without photos)
* KML

All seem to produce bad KML:
* GpsPrune: "No coordinate information found in the file"
* Google Earth: Shows the date and the type ("walking"), but no track

Running OpenTracks 4.1.1 on Android 12

Is this a known issue?

Dave Millman
Ottawa, Canada

**Comment by dennisguse** [comment, marked as answer]
Best guess: we upgraded from KML 2.2 to 2.3 
And there are lot of (open-source) tools that do not yet support KML 2.3 (released in 2016).
Would recommend to open a bug with them.

PS that doesn't mean that there are no bugs in OpenTracks KML exporter though ;)

**Reply by dmillman** [reply]
Thanks Dennis!
